dcos/http.py

```diff
@@ -149,6 +153,7 @@ def request(method,
 
     toml_config = config.get_config()
     auth_token = config.get_config_val("core.dcos_acs_token", toml_config)
+    auth_token_auto_login = config.get_config_val("core.dcos_auto_login", toml_config)
```

Comment:
lets call this `core.prompt_login`. We should add it to the schema for "core": dcos/data/config-schema/core.json